lib/resty/core/regex.lua

```diff
@@ -153,6 +154,14 @@ local function get_max_regex_cache_size()
     return max_regex_cache_size
 end
 
+function _M.regex_cache_is_empty()
+    if not regex_match_cache then
+        return true
+    else
```

Comment:
Isn't one of this module's guidelines to avoid such branches when the previous one is returning? 
```lua
if not regex_match_cache then
    return true
end

return next(regex_match_cache.hasht) == nil
```